packages/core-contracts/contracts/mocks/utils/ERC20Mock.sol

```diff
@@ -0,0 +1,20 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "../../utils/ERC20.sol";
+
+contract ERC20Mock is ERC20 {
+    constructor(
```

Comment:
```suggestion
    // solhint-disable-next-line no-empty-blocks  
    constructor(
```

Reply:
again when i save the file the linter remove the blank line!